### DISCUSSION TITLE
请问如何使用shell里的环境变量

### DISCUSSION THREAD

**Discussion by lilarcor** [kingToolbox/WindTerm#494] [Q&A]
已经设置了aaa变量，值为bbb，如何呼出aaa呢

**Comment by kingToolbox** [comment]
Sorry for the late reply.

In `Windows`, you can invoke the environment variable `aaa` via `%aaa%`, such as executing the command `cmd /c ... %aaa%`. And in `Linux` and `MacOS`, you can do this via `$aaa`.

**Reply by lilarcor** [reply]
thanks for your reply. As I tested, it works with system "cmd", but not for system "powershell". 
<img width="811" alt="image" src="https://user-images.githubusercontent.com/9455380/161358002-ab70770b-6df4-4ee1-b87d-daf18cbf43d5.png">
